modules/govuk/manifests/apps/email_alert_api/sidekiq_queue_check.pp

```diff
@@ -0,0 +1,37 @@
+# == Define: govuk::apps::email_alert_api::sidekiq_queue_checks
+#
+# Creates an Icinga check which checks Graphite for activity
+# on Sidekiq queues
+#
+# === Parameters
+#
+# [*ensure*]
+#   Whether to enable the check.
+#   Default: present
+#
+# [*title*]
+#   The name of the sidekiq queue to check.
+#
+# [*size_warning*]
+#   The number of messages on a queue to trigger a warning.
+#
+# [*size_critical*]
+#   The number of messages on a queue to trigger a critical alert.
+#
+define govuk::apps::email_alert_api::sidekiq_queue_check(
```

Comment:
Are we thinking that the latency and retry set checks will be added into this definition? Just wondering because the name is quite generic.

Reply:
yes, that is the plan